### ISSUE TITLE
Make PBES2 more accessible

### ISSUE THREAD

**Issue by louisabraham** [Legrandin/pycryptodome#398]
Is there a reason why the PBES2 API is hidden in a `_PBES.py` file and not documented?
I think it's great and should be more visible.

**Comment by jneb** [comment]
Yes, please! I was trying to decode a PKCS8 blob (inside a PKCS12 blob, but that's another story...) and I got an error that it didn't recognize the PBES2. Only when looking for an issue, I found that the PBES2 is there at all...
